doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -200,6 +200,7 @@ Bluetooth samples
 * :ref:`peripheral_hids_mouse` and :ref:`central_uart` samples now come with the :ref:`ble_rpc_host` child image configuration overlay.
   The overlay shows how to configure an application running a serialized Bluetooth Low Energy Host.
   These samples run out the box with the :ref:`ble_rpc` library.
+* Added :ref:`central_and_peripheral_hrs` sample.
 
 Bluetooth mesh samples
```

Comment:
@greg-fer this is being added under samples, but should probably get a separate section under nRF5, right?

Reply:
We don't have nRF5 section anymore. We try to mirror code and doc sections in the release notes changelog. So BT mesh samples should go under Samples / Bluetooth samples / BT mesh samples.

Reply:
Oh. Does that mean that changes to the Bluetooth Mesh subsystem actually should go under samples, like Ingar has done here, or do we need to add some new section for that? ...or should those just not be added anymore?

Reply:
Would it be possible to add those to Protocols / BT mesh section? That section is meant for describing bigger protocol-related changes or changes that do not fit elsewhere.

Reply:
I think so. @Balaklaka ?

Reply:
I have no idea where things should go with this layout. Does it fit under Protocols or Libraries? I put it here, so all mesh related things was one place and kept together.

Reply:
You can add generic entries under protocol that link to the libraries and samples sections for BT mesh with detailed changes. With the old layout, we had sections for hardware, which in the long term would mean you would have to add the same entry three or four times under each hardware section.